src/Core_Sideloader/Core.Sideloader.cs

```diff
@@ -229,12 +241,15 @@ private void LoadModsFromDirectories(params string[] modDirectories)
             var failedPaths = allMods.Except(Archives.Select(x => x.Name));
             var failedStrings = failedPaths.Select(GetRelativeArchiveDir).ToArray();
             if (failedStrings.Length > 0)
-                Logger.LogWarning("Could not load " + failedStrings.Length + " mods, see previous warnings for more information. File names of skipped archives:\n" + string.Join(" | ", failedStrings));
+                Logger.LogWarning("Could not load " + failedStrings.Length + " mods, see previous warnings for more information. File names of skipped archives:\n" +
+                                  string.Join(" | ", failedStrings));
         }
 
         private void LoadAllLists(ZipFile arc, Manifest manifest)
         {
             List<ZipEntry> BoneList = new List<ZipEntry>();
+            bool mainGame = Application.productName == Constants.GameProcessName;
```

Comment:
This will only work in Japanese version of the game since the Steam version has a different name. Compare to the studio name instead.

Reply:
Thanks for the tip. I'll keep that in my mind.

Reply:
https://github.com/IllusionMods/BepisPlugins/pull/122/commits/a57e34b7c5f0c984a134ea1e0023d8c6bfee7d86